filebeat/tests/system/test_container.py

```diff
@@ -0,0 +1,66 @@
+from filebeat import BaseTest
+import socket
+import os
+
+
+class Test(BaseTest):
+    """
+    Test filebeat with the container input
+    """
+
+    def test_container_input(self):
+        """
+        Test container input
+        """
+        input_raw = """
+- type: container
+  paths:
+    - {}/logs/*.log
+"""
+        self.render_config_template(
+            input_raw=input_raw.format(os.path.abspath(self.working_dir)),
+            inputs=False,
+        )
+
+        os.mkdir(self.working_dir + "/logs/")
+        self.copy_files(["logs/docker.log"],
+                        target_dir="logs")
+
+        filebeat = self.start_beat()
+
+        self.wait_until(lambda: self.output_count(lambda x: x >= 21))
```

Comment:
I think this could be 
```suggestion
        self.wait_until(lambda: self.output_has(lines=21))
```